src/frontend/screens/ObservationCreate/index.tsx

```diff
@@ -169,12 +211,68 @@ export const ObservationCreate = ({
     value,
   ]);
 
+  const checkAccuracyAndLocation = React.useCallback(() => {
+    const confirmationOptions: AlertButton[] = [
+      {
+        text: formatMessage(m.saveAnyway),
+        onPress: createObservation,
+        style: 'default',
+      },
+      {
+        text: formatMessage(m.manualEntry),
+        onPress: () => navigation.navigate('ManualGpsScreen'),
+        style: 'cancel',
+      },
+      {
+        text: formatMessage(m.keepWaiting),
+        onPress: () => {},
+      },
+    ];
+
+    if (!value) {
+      noop();
```

Comment:
This is not necessary

```suggestion
      noop();
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/702/commits/9ed6b36b9c35fd46ccb358820f0d0980cfc3aab7